### PR TITLE
Text generation

### PR DESCRIPTION
**Pull request by yashk2810**
Text Generation with Tensorflow Eager Execution

### CODE REVIEW

**Comment by yashk2810** [comment]
@alextp @MarkDaoust @lamberta @random-forests @asimshankar

**Comment by lamberta** [comment]
Hi Yash, can you coordinate these notebooks with Josh for the 1.10 release?

**Comment by yashk2810** [comment]
@lamberta I did ask Josh and he told me to put these here for now and then for 1.10, we can move it to wherever the folder is

File: tensorflow/contrib/eager/python/examples/generative_examples/text_generation.ipynb [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "# Import TensorFlow >= 1.9 and enable eager execution\n",
+        "import tensorflow as tf\n",
+        "\n",
+        "# Note: Once you enable the eager execution mode, it cannot be disabled. \n",
```
Comment (alextp) [inline tensorflow/contrib/eager/python/examples/generative_examples/text_generation.ipynb]: Why would the user want to restart the runtime? If you don't have a good answer I'd suggest removing this warning

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/eager/python/examples/generative_examples/text_generation.ipynb (hunk 1 above)
Comment (yashk2810) [reply tensorflow/contrib/eager/python/examples/generative_examples/text_generation.ipynb]: Done

**Review by alextp — APPROVED** [review]

**Comment by alextp** [comment]
Now let's wait for the hold on merges to be lifted to merge this.

**Pull request merged**